packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/SettingsDropdown/index.tsx

```diff
@@ -0,0 +1,113 @@
+import { Translation } from '@suite-components';
+import { Dropdown, DropdownRef } from '@trezor/components';
+import React, { useRef, useState } from 'react';
+import styled from 'styled-components';
+import ActionItem from '../ActionItem';
+import { useActions } from '@suite-hooks';
+import * as routerActions from '@suite-actions/routerActions';
+
+const Wrapper = styled.div<Pick<Props, 'marginLeft'>>`
+    ${props => props.marginLeft && `margin-left: 8px`};
+    position: relative;
+`;
+
+interface Props {
+    marginLeft?: boolean;
+    isActive?: boolean;
+}
+
+const SettingsDropdown = (props: Props) => {
+    const [open, setOpen] = useState(false);
+    const dropdownRef = useRef<DropdownRef>();
+
+    const { goto } = useActions({
+        goto: routerActions.goto,
+    });
+
+    return (
+        <Wrapper {...props}>
+            <Dropdown
+                onToggle={() => setOpen(!open)}
+                ref={dropdownRef}
+                alignMenu="right"
+                offset={34}
+                horizontalPadding={10}
+                topPadding={0}
+                minWidth={230}
+                masterLink={{
+                    callback: () => {
+                        goto('settings-index');
+                    },
```

Comment:
```suggestion
                    callback: () => goto('settings-index'),
```

Reply:
I would prefer the current version with block and no return value. See https://github.com/trezor/trezor-suite/pull/3794/files/49d0e0e35470c138f74ef203ff0e37422cde0c83#r640705764

Reply:
I'd also rather have it used same way as everywhere else to be consistent (we need default return there so the dropdown is closed after click)